sonar-plugin/api/src/main/java/org/sonar/plugins/javascript/api/CustomRuleRepository.java

```diff
@@ -32,8 +36,27 @@
 @Deprecated(since = "6.0")
 public interface CustomRuleRepository {
   enum Language {
-    JAVASCRIPT,
-    TYPESCRIPT,
+    JAVASCRIPT("js"),
+    TYPESCRIPT("ts");
+
+    private static final Map<String, Language> stringMap = Arrays.stream(values()).collect(
+      Collectors.toMap(Enum::toString, Function.identity())
+    );
```

Comment:
holdup, this is to create the map: { 'js'-> JAVASCIPT, 'ts'-> TYPESCRIPT }. There must be an easier way. Perhaps something like:
```suggestion
    private static final Map<String, Language> stringMap = Map.of(
      "js", Language.JAVASCRIPT,
      "ts", Language.TYPESCRIPT
    );
```